src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,366 @@
+     */
+    public void noNumberToMark() {
+        System.out.println(divider);
+        System.out.println("Please provide a task number to mark as done");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles case where task is already marked.
+     */
+    public void alreadyMarked() {
+        System.out.println(divider);
+        System.out.println("This task is already marked done");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles case where the item to mark does not exist.
+     */
+    public void noItemToMark() {
+        System.out.println(divider);
+        System.out.println("Such an item does not exist");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles improper mark keyword entries.
+     */
+    public void markFormatError() {
+        System.out.println(divider);
+        System.out.println("Invalid command please add a space between " +
+                "mark and the list item you would like to interact with. \n" +
+                "Additionally ensure you have entered a number after mark.");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles unmark keyword and unmarks tasks.
+     *
+     * @param text String to unmark
+     */
+    public void unmark(String text) {
+        System.out.println(divider);
+        System.out.println("Ok, I've marked this task as not done yet \n" +
+                storage.taskList.arrayList.get(Integer.parseInt(text.replace("unmark ", "")) - 1).toString() +
+                "\nNow you have " + storage.taskList.arrayList.size() + " tasks in the list");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles case where unmark number is missing.
+     */
+    public void noNumberToUnmark() {
+        System.out.println(divider);
+        System.out.println("Please provide a task number to mark as not done");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles case where task is already unmarked.
+     */
+    public void alreadyUnmarked() {
+        System.out.println(divider);
+        System.out.println("This task is already marked not done");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles case where item to unmark does not exist.
+     */
+    public void noItemToUnmark() {
+        System.out.println(divider);
+        System.out.println("Such an item does not exist");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles improper unmark keyword entries.
+     */
+    public void unmarkFormatError() {
+        System.out.println(divider);
+        System.out.println("Invalid command please add a space between " +
+                "unmark and the list item you would like to interact with. \n" +
+                "Additionally ensure you have entered a number after unmark.");
+        System.out.println(divider);
+    }
+
+    /**
+     * Handles the help keyword and displays the user guide.
+     */
+    public void help() {
```

Comment:
This method is very well done and it was something that most people would have overlooked , Great Job 👏